packages/ignition/src/app/[lang]/page.tsx

```diff
@@ -18,138 +19,153 @@ export const generateMetadata = ({ params: { lang } }: PropsWithLangParams): Met
+                  action: ScriptAction.DELETE_TYPING,
+                  elementId: "el_0.el_1.el_1.el_1",
+                  from: "icon-red-900-md",
+                  to: "icon-",
+                  skipCommit: true
+                },
+                {
+                  action: ScriptAction.UPDATE_TYPING,
+                  elementId: "el_0.el_1.el_1.el_1",
+                  text: "sky-500"
+                },
+                {
+                  time: "5s",
+                  action: ScriptAction.UPDATE_TYPING,
+                  elementId: "el_0.el_1.el_1.el_1",
+                  text: "-lg"
+                },
+                {
+                  time: "2s",
+                  action: ScriptAction.UPDATE_TYPING,
+                  elementId: "el_0.el_1.el_1.el_1",
+                  text: " dark:icon-white-lg",
+                  delay: 30
+                },
+                {
+                  time: "1s",
+                  action: ScriptAction.UPDATE_TYPING,
+                  elementId: "el_0.el_1.el_1.el_1",
+                  text: " mx-1"
+                },
+                {
+                  time: "30s",
+                  action: ScriptAction.REPLACE_TYPING,
+                  elementId: "el_0.el_1.el_1.el_1",
+                  text: "icon-slate-900-base dark:icon-red-500-base"
+                },
+                {
+                  action: ScriptAction.UPDATE,
+                  elementId: "el_0",
+                  text: "h-32"
+                },
+                {
+                  action: ScriptAction.UPDATE,
+                  elementId: "el_0.el_0",
+                  text: "size-10"
+                },
+                {
+                  action: ScriptAction.RESTART
+                }
+              ]}
+            >
+              <div className="h-32">
+                <RcRocketIcon data-cb-tag="RcRocketIcon" className="size-10" />
+                <div>
+                  <div className="text-slate-900 text-xl xs:text-2xl font-light dark:text-white mt-1 xs:mt-2">
+                    <RocketIconsText data-cb-tag="RocketIconsText" />
+                  </div>
+                  <div className="mr-2 text-sm xs:text-base">
+                    Styling in a way
+                    <RcRocketIcon
+                      data-cb-tag="RcRocketIcon"
+                      className="icon-slate-900-base dark:icon-red-500-base"
+                    />
+                    you&apos;ve never seen before.
+                  </div>
+                  <div className="mt-0.5 text-xs leading-4 xs:leading-6">
+                    A funny way handling icons
+                  </div>
                 </div>
               </div>
-            </div>
-          </AnimatedCodeBlock>
-        </div>
-      </div>
-      <div className="mt-5 md:mt-0 w-full max-w-7xl">
-        <Footer />
+            </AnimatedCodeBlock>
+          </div>
+        </section>
+        <section className="mt-5 md:mt-0 w-full max-w-7xl">
```

Comment:
It's semantic accepted, but I think the html will be more sense not having section here. 

Something like:

```
<body>
  <header></header>
  <main>
      <nav></nav>
      <section></section>
      <section>
         <article></article>
      </section>
   </main>
   <footer></footer>
   ....
```